src/Tribe/Abstract_Attendance_Totals.php

```diff
@@ -50,4 +50,44 @@ public function integrate_with_attendee_screen() {
 	 * Prints an HTML (unordered) list of attendance totals.
 	 */
 	abstract public function print_totals();
+
+	/**
+	 * Get Attendee Total Sold Tooltip
+	 *
+	 * @since TBD
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_total_sold_tooltip() {
+		ob_start();
```

Comment:
Should we move this towards an `admin-view` file?

Reply:
I would like to us continue with the tooltips being in similar setup to the status manager ones on the Order page: https://github.com/moderntribe/event-tickets/blob/master/src/Tribe/Status/Abstract_Commerce.php#L163